webapp/views.py

```diff
@@ -451,6 +453,24 @@ def advantage_view():
         for account in accounts:
             account["contracts"] = advantage.get_account_contracts(account)
 
+            if account is not None:
```

Comment:
Can an account be `None`? I think if one was to be the program would blow up on line `454`

Reply:
Good catch, I removed it